apps/OpenSpace/ext/launcher/src/sgctedit/settingswidget.cpp

```diff
@@ -78,13 +96,75 @@ bool SettingsWidget::vsync() const {
 }
 
 bool SettingsWidget::showUiOnFirstWindow() const {
-    return _showUiOnFirstWindow->isChecked();
+    return (_showUiOnFirstWindow->isChecked() && _showUiOnFirstWindow->isEnabled());
 }
 
 void SettingsWidget::setShowUiOnFirstWindow(bool setUiOnFirstWindow) {
     _showUiOnFirstWindow->setChecked(setUiOnFirstWindow);
 }
 
+void SettingsWidget::setEnableShowUiOnFirstWindowCheckbox(bool enable) {
+    _showUiOnFirstWindow->setEnabled(enable);
+    _firstWindowGraphicsSelection->setEnabled(enable);
+}
+
+int SettingsWidget::graphicsSelectionForShowUiOnFirstWindow() const {
+    return _firstWindowGraphicsSelection->currentIndex();
+}
+
+void SettingsWidget::setGraphicsSelectionForShowUiOnFirstWindow(int selection) {
+    _firstWindowGraphicsSelection->setCurrentIndex(selection);
+}
+
 void SettingsWidget::setVsync(bool enableVsync) {
     _checkBoxVsync->setChecked(enableVsync);
 }
+
+void SettingsWidget::nWindowsDisplayedChanged(int newCount) {
+    constexpr int countOneWindow = 1;
+    constexpr int countTwoWindows = 2;
```

Comment:
```suggestion
    constexpr int CountOneWindow = 1;
    constexpr int CountTwoWindows = 2;

```

Since they are constants